plugins/org.eclipse.papyrus.uml.interaction.model/model/seqd.ecore

```diff
@@ -114,6 +114,20 @@
         </eAnnotations>
       </eParameters>
     </eOperations>
+    <eOperations name="getBottomMostElement" lowerBound="1">
```

Comment:
Bottommost is one word (or, at least, "bottom-most") so the M should be lower case.

Reply:
Good point! I'll fix that in a subsequent commit.